### DISCUSSION TITLE
Why don't you introduce a local-only paid subscription (through a yearly/monthly license), similar to Obsidian?

### DISCUSSION THREAD

**Discussion by s7oev** [Kong/insomnia#6625] [Ideas]
You need to make money, I get that. You also prefer to have a stable, subscription-based income, as opposed to a one-time payoff from your users - I also get that.

Yet why force cloud sync? Couldn't you introduce something similar to [Obsidian's commercial use license](https://obsidian.md/pricing) vs [Obsidian Sync](https://obsidian.md/sync)? According to the first license, you always have to pay to use Obsidian commercially. Additionally, you also have to pay for cloud synchronization - a completely separate feature. Thereby allowing developers to subscribe for Obsidian for local-only use, and decide whether they would like to also pay for cloud-sync, separately.

Most importantly - from your POV - the commercial use license has to be renewed per year, it's not a one-off payment.

Wouldn't this situation be a win-win?

**Comment by subnetmarco** [comment]
An update here https://github.com/Kong/insomnia/discussions/6626
